<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>425</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Duduf Frame Player</string>
  </property>
  <property name="windowIcon">
   <iconset resource="ressources.qrc">
    <normaloff>:/icones/dufrap</normaloff>:/icones/dufrap</iconset>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonTextBesideIcon</enum>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="mainWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="playWidget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>10</number>
       </property>
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item>
        <widget class="QWidget" name="playControls" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="playControlsLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QProgressBar" name="bufferBar">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="value">
             <number>0</number>
            </property>
            <property name="textVisible">
             <bool>true</bool>
            </property>
            <property name="format">
             <string>Buffer : %p%</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentTCLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Rien à lire...</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="durationLabel">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>--:--</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="volumeWidget" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>10</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="zoomButton">
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="maxVisibleItems">
             <number>15</number>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>Auto-&gt;100%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Auto</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>200%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>150%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>75%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12%</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5%</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="fpsBox">
            <property name="currentIndex">
             <number>5</number>
            </property>
            <property name="maxVisibleItems">
             <number>15</number>
            </property>
            <property name="frame">
             <bool>false</bool>
            </property>
            <item>
             <property name="text">
              <string>8 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>12 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>15 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>23.976 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>24 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>25 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>29.97 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>30 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>59.94 fps</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>60 fps</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="statusLabel">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Temps réel</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOuvrir"/>
   <addaction name="separator"/>
   <addaction name="actionStop"/>
   <addaction name="actionImage_pr_c_dente"/>
   <addaction name="actionLecture"/>
   <addaction name="actionImage_Suivante"/>
   <addaction name="actionBoucle"/>
   <addaction name="separator"/>
   <addaction name="actionParam_tres"/>
   <addaction name="actionA_propos"/>
   <addaction name="separator"/>
   <addaction name="actionQuitter"/>
  </widget>
  <action name="actionQuitter">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/quit</normaloff>:/icones/quit</iconset>
   </property>
   <property name="text">
    <string>Quitter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionOuvrir">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/folder_open</normaloff>:/icones/folder_open</iconset>
   </property>
   <property name="text">
    <string>Ouvrir...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionLecture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/play</normaloff>:/icones/play</iconset>
   </property>
   <property name="text">
    <string>Lecture</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/stop</normaloff>:/icones/stop</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Space</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionBoucle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/loop</normaloff>:/icones/loop</iconset>
   </property>
   <property name="text">
    <string>Boucle</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::ApplicationShortcut</enum>
   </property>
  </action>
  <action name="actionImage_Suivante">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/next_frame</normaloff>:/icones/next_frame</iconset>
   </property>
   <property name="text">
    <string>Image Suivante</string>
   </property>
   <property name="iconText">
    <string>Image</string>
   </property>
  </action>
  <action name="actionImage_pr_c_dente">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/prev_frame</normaloff>:/icones/prev_frame</iconset>
   </property>
   <property name="text">
    <string>Image précédente</string>
   </property>
   <property name="iconText">
    <string>Image</string>
   </property>
  </action>
  <action name="actionParam_tres">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/params</normaloff>:/icones/params</iconset>
   </property>
   <property name="text">
    <string>Paramètres</string>
   </property>
  </action>
  <action name="actionA_propos">
   <property name="icon">
    <iconset resource="ressources.qrc">
     <normaloff>:/icones/duduf</normaloff>:/icones/duduf</iconset>
   </property>
   <property name="text">
    <string>A propos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
